build.gradle

```diff
@@ -275,6 +275,21 @@ task generateAudioApi(type: org.openapitools.generator.gradle.plugin.tasks.Gener
   ]
 }
 
+task generateAudioRequestsApi(type: org.openapitools.generator.gradle.plugin.tasks.GenerateTask) {
+  generatorName = "spring"
+  inputSpec = "$rootDir/src/main/resources/openapi/audio-requests.yaml".toString()
+  outputDir = "$buildDir/generated/openapi".toString()
+  apiPackage = "uk.gov.hmcts.darts.audiorequests.api"
+  modelPackage = "uk.gov.hmcts.darts.audiorequests.model"
+
+  skipOperationExample = true
+  configOptions = [
+    dateLibrary  : "java8",
+    interfaceOnly: "true",
+    useTags      : "true"
+  ]
```

Comment:
we might want to add `useSpringBoot3: "true"` so that it generates jakarta imports